editor/plugins/script_text_editor.cpp

```diff
@@ -668,6 +668,10 @@ void ScriptEditor::_update_modified_scripts_for_external_editor(Ref<Script> p_fo
 			script->update_exports();
 		}
 	}
+
+	if (auto_reload_running_scripts) {
+		_live_auto_reload_running_scripts();
```

Comment:
Should only call this if a script has been reloaded and `pending_auto_reload == false`